src/theme/custom/cosmoDark/palette.js

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license.
 
 export default {
-  type: 'dark',
+  type: 'light',
```

Comment:
I'm not sure I understand this change, shouldn't we use the cosmoLight theme (or create a new theme) instead of changing the type of the cosmoDark theme to "light"?